portlayer/attach/messages.go

```diff
@@ -14,17 +14,100 @@
 
 package attach
 
+import "golang.org/x/crypto/ssh"
+
 // All of the messages passed over the ssh channel/global mux are (or will be)
 // defined here.
 
-const (
-	WindowChangeReq = "window-change"
-)
+type Messager interface {
```

Comment:
You're right;  the convention is for single method interfaces.  

Maybe we'll add more. It does serialize/deserialize and store data in messages we pass over ssh.  Actually sending is tricky.  I may add that later.

I don't feel strongly about the name and can change it to `Message`.